### PR TITLE
add cell format

### PR DESCRIPTION
**Pull request by mbrocchieri**

### CODE REVIEW

File: fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java [hunk 1]
```diff
@@ -9,9 +9,8 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```
Comment (aousadou) [inline fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java:12]: Remove * imports

File: fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import java.io.FileInputStream;
```
Comment (aousadou) [inline fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java:20]: Remove useless import

File: fastexcel-writer/src/test/java/org/dhatim/fastexcel/WriteFileTest.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.math.BigDecimal;
+
+public class WriteFileTest {
+
+  @Test
+  void testWrite() throws IOException {
+    try (FileOutputStream os = new FileOutputStream("/home/mathieu/testFormulatWithoutValue.xlsx")) {
```
Comment (aousadou) [inline fastexcel-writer/src/test/java/org/dhatim/fastexcel/WriteFileTest.java:15]: I suggest to remove path to your home and change it with target maven dir or /tmp dir

**Review by aousadou — CHANGES_REQUESTED** [review]

File: fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java (hunk 1 above)
Comment (mbrocchieri) [reply fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/OPCPackage.java:12]: fixed

File: fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java (hunk 2 above)
Comment (mbrocchieri) [reply fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java:20]: fixed

File: fastexcel-writer/src/test/java/org/dhatim/fastexcel/WriteFileTest.java (hunk 3 above)
Comment (mbrocchieri) [reply fastexcel-writer/src/test/java/org/dhatim/fastexcel/WriteFileTest.java:15]: removed

**Comment by ochedru** [comment]
Please revert `<version>` tags to `<version>0-SNAPSHOT</version>`.

**Comment by mbrocchieri** [comment]
<version> tags are set to <version>0-SNAPSHOT</version>.

**Comment by ochedru** [comment]
Thank you for your PR!

**Pull request merged**